Update design.md table Kolibri

NicoleTiede opened the pull request.

NicoleTiede committed:
Update design.md

MarkusTiede committed:
Update design.md

MarkusTiede reviewed (changes requested):
Ich würde die Dateinamen alle kleinschreiben und einem Namensschema entsprechen lassen; zudem sind jetzt mMn zu viele SVGs im Repo.

NicoleTiede committed:
Change Low poly art Table

still need instruction on where to save the hummingbird_painted-versions and on how to downsize pictures
